shoop/themes/classic_gray/templates/classic_gray/plugins/social_media_links.jinja

```diff
@@ -0,0 +1,11 @@
+<section class="social-media-links">
+    <div class="row{% if alignment %} text-{{ alignment }}{% endif %}">
+        {% if topic %}<h1>{{ topic }}</h1>{% endif %}
```

Comment:
This should be `<h2>` instead of `<h1>`. We should not be using more than one `<h1>` in one page and usually that is the page title.